website/docs/cloud-docs/api-docs/run.mdx

```diff
@@ -553,7 +572,21 @@ curl \
       }
     },
     {...}
-  ]
+  ],
+  "links": {
+    "self": "https://app.terraform.io/api/v2/organizations/hashicorp/runs?page[number]=2&page[size]=20",
+    "prev": "https://app.terraform.io/api/v2/organizations/hashicorp/runs?page[number]=1&page[size]=20",
+    "next": "https://app.terraform.io/api/v2/organizations/hashicorp/runs?page[number]=3&page[size]=20"
+  },
+  "meta": {
+    "pagination": {
+      "current-page": 1,
+      "next-page": 2,
+      "prev-page": null,
+      "page-size": 20,
+      "page-count": 20
```

Comment:
We should remove this one, right? It's not part of the response

Reply:
Ah good catch, included 👍 https://github.com/hashicorp/terraform-docs-common/pull/770/commits/2b7e2ae81db72296bc36e67abe3f7935104d0510